week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,69 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+
+const veryPositive = ["It is certain.",
+"It is decidedly so.",
+"Without a doubt.",
+"Yes - definitely.",
+"You may rely on it."]
+
+const positive = ["As I see it, yes.",
+"Most likely.",
+"Outlook good.",
+"Yes.",
+"Signs point to yes."]
+
+const negative = ["Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again."
+]
+
+const veryNegative = ["Don't count on it.",
+"My reply is no.",
+"My sources say no.",
+"Outlook not so good.",
+"Very doubtful."]
+
+
+function shakeBall() {
+
+  console.log("The ball has shaken!");
+
+  const answers = veryPositive.concat(positive).concat(negative).concat(veryNegative);
+
+  let answer = answers[Math.floor(Math.random() * Math.floor(19))]
```

Comment:
Why `19`?

Reply:
answers length is 20, so it is from 0 to 19